### ISSUE TITLE
Change Histogram to Barplot or vice versa

### ISSUE THREAD

**Issue by upendrak** [datacarpentry/R-genomics#28]
In the[ 5th chapter](http://www.datacarpentry.org/R-genomics/05-data-visualization.html), i see that even though the section starts with Histogram, the plot type was consistently called as `geom_bar`. I know that there is not much difference between barplot and histogram in ggplot. So either the section heading needs to changed from Histogram to Barplots or `geom_bar` needs to replaced with `geom_histogram` for consistency purposes.

**Comment by alexgraehl** [comment]
Good point! Just modified this to change everything to "geom_histogram"... setting up a pull request now.

**Comment by alexgraehl** [comment]
Just submitted a pull request to address this: https://github.com/datacarpentry/R-genomics/pull/37
